### DISCUSSION TITLE
Trying to send Item discount on view_cart event

### DISCUSSION THREAD

**Discussion by adspinto** [invertase/react-native-firebase#5398] [Q&A]
We are trying to use logEvent to send data to analytics where we have to send items discount and it is not showing.

The part of the code where we send the event
![image](https://user-images.githubusercontent.com/33551553/121068753-073d8100-c7a3-11eb-905f-bc177b8551d9.png)

The result in debug mode:
![image](https://user-images.githubusercontent.com/33551553/121068590-db220000-c7a2-11eb-854e-5185129b29e8.png)

Anyone else has this problem?

**Comment by mikehardy** [comment]
This may be interesting: https://github.com/invertase/react-native-firebase/issues/4018

That one resolved to an upstream issue (that is, either on the firebase backend of the underlying firebase-android-sdk / firebase-ios-sdk. 

I would recommend putting together a quickstart to see if it still happens while using the native SDKs raw - the quickstarts make it pretty easy, e.g. https://github.com/firebase/quickstart-android/tree/master/analytics 

My hunch is that you will still observe the unwanted behavior while using the native SDK, and this won't be an issue that we can fix here. You may also try reaching out on stackoverflow as the Firebase folks hang out there for support sometimes, but you will need to have an example using their SDKs (directly - not via react-native-firebase) to demonstrate the issue
